.travis.yml

```diff
@@ -65,6 +72,15 @@ matrix:
           services: [docker]
           env: BUILD=tests,wheels,release
 
+        - os: linux
```

Comment:
waiting https://travis-ci.org/MagicStack/uvloop/builds/400060588 to finish

may I suggest using **1** `CI` that support `OSX` / `Linux` / `Windows` (instead of **2**)
I've found https://cirrus-ci.org/

Reply:
Seems that Cirrus CI is just a couple of days old. We'll wait until it's mature before spending any time trying to integrate it.